src/strategies/erc1155-all-balances-of/index.ts

```diff
@@ -0,0 +1,56 @@
+import { subgraphRequest } from '../../utils';
+
+export const author = 'fragosti';
+export const version = '0.1.0';
+
+export const SUBGRAPH_URL = {
+  '1': 'https://api.thegraph.com/subgraphs/name/alexvorobiov/eip1155subgraph'
+};
+
+export async function strategy(
+  _space,
+  network,
+  _provider,
+  addresses,
+  options,
+  _snapshot
+) {
+  const eip1155OwnersParams = {
```

Comment:
The subgraph query should include the snapshot block number too, here is an example https://github.com/snapshot-labs/snapshot.js/blob/master/src/strategies/ens-domains-owned/index.ts#L45-L48